src/bindings/js/node/src/node_wrap.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <napi.h>
+#include <ov/node.hpp>
+
+#include <openvino/openvino.hpp>
+
+class NodeWrap : public Napi::ObjectWrap<NodeWrap> {
+public:
+    /**
+     * constructor
+     * @brief Constructs NodeWrap from the Napi::CallbackInfo.
+     * @param info contains passed arguments. Can be empty.
+     * @param node pointer to ov::Node object.
+     */
+    NodeWrap(const Napi::CallbackInfo& info, ov::Node* node);
```

Comment:
```suggestion
    NodeWrap(const Napi::CallbackInfo& info, const ov::Node* node);
```
or 

```suggestion
    NodeWrap(const Napi::CallbackInfo& info, const ov::Node& node);
```
to not allow pass null pointer and initialize `node_(&node)`.

@almilosz 
Is `ov::Node*` parameter is required? 
Other wrappers initialize the internal member from NAPI info or make default initialization and have setter to set it.

Reply:
you're right, & makes more sense here since node_ should never be nullptr. I'll update it, thanks for the suggestion:)